cmd/admin-info.go

```diff
@@ -187,31 +187,35 @@ func mainAdminInfo(ctx *cli.Context) error {
 	// If the error is not nil and not unrecognizable, just print it and exit
 	fatalIf(probe.NewError(e), "Cannot get service status.")
 
-	// Construct the backend status
-	storageInfo := backendStatus{
-		Total: serverInfo.StorageInfo.Total,
-		Free:  serverInfo.StorageInfo.Free,
-	}
-	if serverInfo.StorageInfo.Backend.Type == madmin.Erasure {
-		storageInfo.Backend = xlBackend{
-			Type:         erasureType,
-			OnlineDisks:  serverInfo.StorageInfo.Backend.OnlineDisks,
-			OfflineDisks: serverInfo.StorageInfo.Backend.OfflineDisks,
+	for _, serverInfo := range serversInfo {
```

Comment:
![screenshot from 2017-05-13 20-37-25](https://cloud.githubusercontent.com/assets/283197/26028615/98c08b8a-3824-11e7-9296-50ab07eb32d1.png)

This is an example of output but I didn't update the PR until another PR in minio/minio gets merged.